packages/shared/src/domain/patient.ts

```diff
@@ -2,7 +2,7 @@ import { z } from "zod";
 
 export const patientSchema = z.object({
   id: z.string(),
-  externalId: z.string().optional(),
+  externalId: z.string().optional().nullable(),
```

Comment:
i _think_ optional().nullable() can be simplified to nullish()?

Reply:
Nice didn't know about that one, addressed!